acc/kernels/__init__.py

```diff
@@ -45,4 +45,13 @@ def simplepowderdiffraction_scattering_coefficient(neutron):
             return scattering_coefficient(neutron, ucvol, Npeaks, q_v, my_s_v2)
         return simplepowderdiffraction_scatter, simplepowderdiffraction_scattering_coefficient
 
+    def onConstantQEKernel(self, kernel):
+        from .constant_qe import S
```

Comment:
@ckendrick could you add a line here to obtain `sigma`, similar to the implementation for isotropic kernel